lib/features/email/presentation/extensions/presentation_email_extension.dart

```diff
@@ -71,20 +79,34 @@ extension PresentationEmailExtension on PresentationEmail {
           .removeDuplicateEmails()
           .withoutMe(userName);
 
-        return Tuple3(
+        return Tuple4(
           listToAddressWithoutUsername,
           listCcAddressWithoutUsername,
           listBccAddressWithoutUsername,
+          []
+        );
+      case EmailActionType.editDraft:
+        final listToAddressWithoutUsername = newToAddress.withoutMe(userName);
+        final listCcAddressWithoutUsername = newCcAddress.withoutMe(userName);
+        final listBccAddressWithoutUsername = newBccAddress.withoutMe(userName);
+        final listReplyToAddressWithoutUsername = newReplyToAddress;
+
+        return Tuple4(
+          listToAddressWithoutUsername,
+          listCcAddressWithoutUsername,
+          listBccAddressWithoutUsername,
+          listReplyToAddressWithoutUsername
         );
```

Comment:
With Draft email we should keep the To, Cc, Bcc, ReplyTo fields intact.
```suggestion
        return Tuple4(
          newToAddress,
          newCcAddress,
          newBccAddress,
          newReplyToAddress,
        );
```

Reply:
done !